autoload/neoformat.vim

```diff
@@ -231,8 +227,16 @@ function! s:generate_cmd(definition, filetype) abort
 
     let filename = expand('%:t')
 
+    let tmp_dir = has('win32') ?
+                \ expand('$TEMP/neoformat') :
+                \ expand('$TMPDIR/neoformat')
+
+    if !isdirectory(tmp_dir)
+        call mkdir(tmp_dir, 'p')
+    endif
+
     if get(a:definition, 'replace', 0)
-        let path = !using_stdin ? '/tmp/neoformat/' . fnameescape(filename) : ''
+        let path = !using_stdin ? expand(tmp_dir . '/' . fnameescape(filename)) : ''
```

Comment:
yes
if `&shellslash == 0` , `expand()` will convert `/` to `\`
else will convert to `/`